flake.nix

```diff
@@ -27,6 +27,9 @@
   inputs.crane.url = "github:ipetkov/crane";
   inputs.crane.inputs.nixpkgs.follows = "nixpkgs";
 
+  inputs.fenix.url = "github:nix-community/fenix";
+  # inputs.fenix.inputs.nixpkgs.follows = "nixpkgs";
```

Comment:
nonblocking question: how come we're not doing the follows? Drop the comment?

Reply:
We use it as an overlay so setting the follow or not does not change much, we should set it anyway though I think to avoid inflating the lock file.